mint-auditor/src/db/mod.rs

```diff
@@ -85,13 +80,7 @@ impl MintAuditorDb {
         block: &Block,
         block_contents: &BlockContents,
         ledger_db: &LedgerDB,
-    ) -> Result<
-        (
-            crate::db::block_audit_data::BlockAuditData,
-            HashMap<TokenId, u64>,
-        ),
-        Error,
-    > {
+    ) -> Result<(crate::db::models::BlockAuditData, HashMap<TokenId, u64>), Error> {
```

Comment:
This qualified path seems unnecessary.
```suggestion
    ) -> Result<(BlockAuditData, HashMap<TokenId, u64>), Error> {
```